src/controllers/scoresController.ts

```diff
@@ -48,6 +49,10 @@ async function getScoresFromDB(gameIndex: number): Promise<object[]> {
       ORDER BY score DESC, datetime DESC;`,
       [gameIndex]
     );
+    res.rows.forEach(row => {
```

Comment:
Make this a function, maybe in `util`?

Reply:
Done https://github.com/knative-portability/Kubercade/commit/7488a5b34816ae2f301f453a9bc4cc02341f5695